package.json

```diff
@@ -3,17 +3,17 @@
   "version": "5.4.0",
   "description": "Zip it and ship it",
   "main": "./dist/main.js",
+  "types": "./dist/main.d.ts",
   "bin": {
     "zip-it-and-ship-it": "dist/bin.js"
   },
   "files": [
-    "dist/**/*.js",
-    "dist/**/*.d.ts"
+    "dist/**/*.{js,ts}"
   ],
   "scripts": {
     "prepublishOnly": "npm ci && npm test",
     "prepack": "npm run build",
-    "build": "tsc",
+    "build": "rimraf dist && tsc",
```

Comment:
This change is probably worth having it in all your TS projects to make sure the dist directory doesn't contain any leftovers (probably doesn't apply for automated publishes where the dist dir is clean already).

Reply:
We are publishing via CI, so pulling in a new dev dependency is unneccessary in this case.

Reply:
Cool, I'll revert it I guess :)